### DISCUSSION TITLE
카테고리의 다른 글 스타일 적용 안됨

### DISCUSSION THREAD

**Discussion by Bonguk** [pronist/hELLO#115] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: Windows 11
- 브라우저: chrome 103.0.5060.53(공식 빌드) (64비트)
- 버전: [e.g. 22]
- 스킨 버전 (스킨 버전은 index.xml 에서 확인): 3.6.1

**설명**
카테고리의 다른 글 스타일 적용 안됨
본문 글 작성 후 글 내용 중 하단 "카테고리의 다른 글" 부분이 스타일 적용이 안됨

**발생 과정**
처음 블로그 스타일 적용 후 글을 작성 하고 난 뒤 

**스크린샷 및 링크**
![image](https://user-images.githubusercontent.com/47359380/176205560-f2b00002-317b-4556-ae3c-c9ae3d00f175.png)

**Comment by pronist** [comment, marked as answer]
@Bonguk 안녕하세요. 이 문제는 **티스토리 스킨 설정 - 플러그인 - 카테고리의 다른 글**을 `Off` 로 변경하시길 바랍니다.
